assignments/objects.js

```diff
@@ -19,27 +19,80 @@ const example = {
 
 // Write your intern objects here:
 
+const employeeOne = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+  }
+
+const employeeTwo = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
+  
+  }
+const employeeThree = {
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+  
+  }
+
+const employeeFour = {
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+  
+  }
+
+  const employeeFive = {
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F"
+  
+  }
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
 
+console.log(employeeOne["name"]);
+
 // Kennan's ID
 
+console.log(employeeTwo["id"]);
+
 // Keven's email
 
+console.log(employeeThree["email"]);
+
 // Gannie's name
 
+console.log(employeeFour["name"]);
+
 // Antonietta's Gender
 
+console.log(employeeFive["gender"]);
+
+
 // ==== Challenge 3: Object Methods ==== 
```

Comment:
In this Challenge, it wants you to add a method into the kennan(employeeTwo) you can assigned a name into an object by doing employeeTwo.speak then assign a function to it. 
employeeTwo.speak = function() {
 (Your block of code here)
}